src/modules/add_task.js

```diff
@@ -0,0 +1,22 @@
+const input = document.querySelector('.input');
+
+const addToLists = (tasks) => {
+  input.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      if (!input.value) {
+        alert('Please enter a task');
+        return;
+      }
+      const eachTask = {
+        description: input.value,
+        completed: false,
+        index: tasks.length + 1,
+      };
+
```

Comment:
- [ ] Actually when we add some todos, delete one and add another one as you can see some todos comes out with the same index. This creates issues when we now delete one of the todos with same index. please kindly consider and fix it
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/37901469/192895853-35d57b42-e66c-4dea-a52e-8adc73fd7618.png">